# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wmacpi/makfile-install.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-07-19 18:54:03.112779304 +0200
+++ ./Makefile	2003-07-19 18:55:44.165416968 +0200
@@ -1,6 +1,9 @@
 # set options.  pick one, acpi or apm.  comment out the other one.  don't
 # uncomment both, bad things will happen :)
 
+PREFIX=/usr
+DESTBIN=$(PREFIX)/bin
+
 OPT	= -O3 -DACPI
 #OPT	= -O3 -DAPM
 
@@ -21,5 +24,12 @@
 
 wmacpi:	$(OBJS)
 
+install:
+	install -m 755 -d $(DESTBIN)
+	install -m 755 wmacpi $(DESTBIN)
+
+uninstall:
+	rm -f $(DESTBIN)/wmacpi
+
 clean:
 	rm -f *.o *~ wmacpi trace *.out *.bb *.bbg
